src/mmw/js/src/modeling/modificationConfig.json

```diff
@@ -0,0 +1,98 @@
+{
+    "chaparral": {
+        "name": "Chaparral",
+        "summary": "This category includes areas dominated by shrubs; less than 5 meters tall with shrub canopy typically greater than 20% of total vegetation. This class includes true shrubs, young trees in an early successional stage or trees stunted from environmental conditions.",
+        "color": "#e2d9b5"
+    },
+    "commercial": {
+        "name": "Commercial",
+        "summary": "This category includes any area that is entirely paved and is not residential. It includes roads, railroads, shopping centers, and factories.",
+        "color": "#40c2a8"
+    },
+    "desert": {
+        "name": "Desert",
+        "summary": "This category includes areas of bedrock, desert pavement, scarps, talus, slides, volcanic material, glacial debris, sand dunes, strip mines, gravel pits and other accumulations of earthen material. Generally, vegetation accounts for less than 15% of total cover.",
+        "color": "#f1edd9"
+    },
+    "forest": {
+        "name": "Forest",
+        "summary": "This category includes areas that are covered by vegetation with more than 20% of the land surface covered by trees that are taller than 5 meters.",
+        "color": "#53e9b4"
+    },
+    "grassland": {
+        "name": "Grassland",
+        "summary": "This category includes areas where grasses and weedy (herbaceous) plants cover more than 80% of the ground area. These areas are not intensively managed but may be used for grazing animals.",
+        "color": "#54d2d0"
+    },
+    "hir": {
+        "name": "High Intensity Residential",
+        "shortName": "HIR",
+        "summary": "This category includes areas with apartment complexes or row houses. The amount of area that is covered by hard surfaces like roads, driveways and roofs is between 80% and 100%. No more than 20% of the area is covered with vegetation.",
+        "color": "#39afa1"
+    },
+    "lir": {
```

Comment:
I like how almost all the modification configuration is now present in one file. Maybe someday we'll be able to get the patterns in here too. But this is great for now.